src/components/widget/Link.js

```diff
@@ -1,11 +1,9 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 
-class Link extends Component {
-  constructor(props) {
-    super(props);
-  }
-
-  handleClick = url => {
+export default class Link extends PureComponent {
+  handleClick = () => {
+    const { widgetData } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['widgetData' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4689744)